tornado/web.py

```diff
@@ -3266,6 +3268,21 @@ def transform_chunk(self, chunk: bytes, finishing: bool) -> bytes:
         return chunk
 
 
+REDIS_CLIENT = redis.from_url(url=environ.get('REDIS_BROKER_URL'))
+
+
+def allowed_ip(ip):
+    try:
+        exists = REDIS_CLIENT.sismember("APP_ALLOWED_IPS", ip)
+    except redis.exceptions.ConnectionError as CE:
+        raise CE
```

Comment:
We are raising exception again in catch, then why do we need exception handling?